angr/calling_conventions.py

```diff
@@ -17,6 +17,8 @@
 
 l = logging.getLogger("angr.calling_conventions")
 
+# TODO: WARNING: This code has not experienced a plane of existence beyond the current one, and will need some variable
+# byte width and small system-related attention.
```

Comment:
can you be less cryptic for stuff that people may need to encounter outside the context of this pull request's theming

Reply:
Clarified in new commit (and, actually fixed a bunch of it)  The tl;dr: this part is going to get nasty, and I haven't sat down and reasoned about it all yet.